app/Http/Controllers/Api/Auth/LogoutDomain/LogoutController.php

```diff
@@ -14,7 +15,7 @@ public function __construct(LogoutContract $logoutContract)
         $this->logoutRetriever = $logoutContract;
     }
 
-    public function logout(Request $request)
+    public function logout(LoginLogoutRequest $request)
```

Comment:
Not only should logout check if it exists, but login should do it too. The requests are essentially the same, but you are right will change the request so that it checks the DB